tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -965,13 +965,14 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args):
                 args['onnx_opset_version'] = None
 
                 try:
-                    ov_model = _convert(cli_parser, framework, args)
+                    ov_model, argv = _convert(cli_parser, framework, args)
                 except Exception as e:
                     remove_tmp_onnx_model(out_dir)
                     raise e
 
                 remove_tmp_onnx_model(out_dir)
-                return ov_model
+
+                return ov_model, argv
```

Comment:
This does not change the logic of method. 
In previous version ov_model contained tuple with (ov_model, argv) and this tuple was returned as result.
Renaming looks good, but it is not the root cause of the issue.